src/tokenList.json

```diff
@@ -102,5 +102,43 @@
     "website": "https://www.synthetix.io/ ",
     "description": "Tracks the price of a single US Dollar (USD). This Synth always remains constant at 1.",
     "rinkeby_faucet": "https://rinkeby.etherscan.io/address/0xDFd3CAde426Fc6A014fd2e0E1Cb3158F972A1A0D. Instructions: https://github.com/Synthetixio/synthetix/issues/382#issuecomment-579518740"
+  },
+  {
+    "id": 10,
+    "name": "Synthetix Network",
+    "symbol": "SNX",
+    "decimals": 18,
+    "addressByNetwork": {
+      "1": "0xC011A72400E58ecD99Ee497CF89E3775d4bd732F",
+      "4": "0x322A3346bf24363f451164d96A5b5cd5A7F4c337"
+    },
+    "website": "https://www.synthetix.io",
+    "description": "The Synthetix Network Token (SNX) gets staked as collateral to back Synths and entitles stakers to receive fees generated by Synth trades on Synthetix.Exchange",
+    "rinkeby_faucet": "https://rinkeby.etherscan.io/address/0xDFd3CAde426Fc6A014fd2e0E1Cb3158F972A1A0D. Instructions: https://github.com/Synthetixio/synthetix/issues/382#issuecomment-579518740"
+  },
+  {
+    "id": 11,
+    "name": "Synth sETH",
+    "symbol": "sETH",
+    "decimals": 18,
+    "addressByNetwork": {
+      "1": "0x5e74C9036fb86BD7eCdcb084a0673EFc32eA31cb"
```

Comment:
No rinkeby address?

Reply:
Nup, I didn't find. I think we don't need to add every token to Rinkeby. With having some for testing would suffice imo

Reply:
Found it! 

https://developer.synthetix.io/api/docs/deployed-contracts.html#rinkeby-contracts
https://rinkeby.etherscan.io/address/0xA83AbFdC9E8Ee990C3C6C0f56a4B06e0faAd583C

Reply:
Thanks!!!!!!!